web/src/OverviewActionBar.tsx

```diff
@@ -251,36 +260,56 @@ export let ButtonRightPill = styled(ButtonRoot)`
   border-radius: 0 4px 4px 0;
 `
 
+/**
+ * Style todos:
+ * - Remove blue border on hover when field has .Mui-error
+ * - Error helper text should flex-direction: row at large screen sizes
+ * - Error helper text should have margin-left, margin-right: 0; on laptop below sizes
+ * - OverviewActionBar flex container should align-items: flex-start when error helper text is visible
+ * - (bonus) Buttons and input should have padding on small screen sizes (when stacked)
+ */
 const FilterTermTextField = styled(TextField)`
   & .MuiOutlinedInput-root {
-    border-radius: ${SizeUnit(0.125)};
-    border: 1px solid ${Color.grayLighter};
     background-color: ${Color.grayDark};
-    transition: border-color ${AnimDuration.default} ease;
-    width: ${SizeUnit(8.125)};
+    width: ${SizeUnit(9)};
 
-    &:hover {
-      border-color: ${Color.blue};
-    }
     & fieldset {
-      border-color: 1px solid ${Color.grayLighter};
+      transition: border-color ${AnimDuration.default} ease;
+      border-radius: ${SizeUnit(0.125)};
+      border: 1px solid ${Color.grayLighter};
     }
     &:hover fieldset {
-      border: 1px solid ${Color.grayLighter};
+      border: 1px solid ${Color.blue};
     }
     & .Mui-focused fieldset {
       border: 1px solid ${Color.grayLighter};
+      outline: none;
     }
     & .MuiOutlinedInput-input {
       padding: ${SizeUnit(0.2)};
     }
   }
 
   & .MuiInputBase-input {
-    font-family: ${Font.monospace};
     color: ${Color.gray7};
+    font-family: ${Font.monospace};
     font-size: ${FontSize.small};
   }
+
+  & .MuiFormHelperText-root {
```

Comment:
fyi, i think the & bit here is redundant?

Reply:
Ah yes! I ended up moving these styles to a new selector, but ty for pointing out.